recipes/enchant/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+import functools
+
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class EnchantConan(ConanFile):
+    name = "enchant"
+    description = (
+        "Enchant aims to provide a simple but comprehensive abstraction for "
+        "dealing with different spell checking libraries in a consistent way"
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://abiword.github.io/enchant/"
+    topics = "enchant", "spell", "spell-check"
+    license = "LGPL-2.1-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+    requires = "glib/2.71.3", "hunspell/1.7.0"
+    exports_sources = \
+        "CMakeLists.txt", "compile-checks.cmake", "configmake.h", "patches/*"
+
+    def validate(self):
+        if self.settings.os == "Macos":
+            raise ConanInvalidConfiguration(
+                "enchant is not supported on macOS yet"
```

Comment:
I don't have Apple hardware to fix things on, so I can't do anything but disable the recipe there.

Reply:
Nevermind, I found a way to address macOS support.